.github/ISSUE_TEMPLATE/release_template.md

```diff
@@ -53,6 +54,49 @@ Bugs & improvements:
 
 - [ ] (1) ...
 
+_____
+
+## Patch Release
+
+### TASKS:
+
+ - [ ] [GIT] Create branch `release/M.m.p` in owncloud/android from `latest`
+ - [ ] [DEV] Update version number and name in build.gradle in owncloudApp module
+ - [ ] [DEV] Update [SBOM](https://cloud.owncloud.com/f/6072870)
+ - [ ] [DIS] Create a folder for the new version like `M.m.p_YYYY-MM-DD` inside the `changelog` folder with the proper fixes
+ - [ ] [DIS] Move the `unreleased` folder from `master` to `changelog` folder, to avoid Calens' conflicts problems
+ - [ ] [DEV] Add release notes replacing `emptyList` with `listOf` and adding inside `ReleaseNote()` with String resources, in `ReleaseNotesViewModel.kt` (in case patch release before OEM)
+ - [ ] [DIS] Check for new screenshots in Play Store / GitHub repo and generate them
+ - [ ] [DIS] Generate final bundle from last commit in owncloud/android
+ - [ ] [DEV] Code Review
+ - [ ] [QA] Test execution
+ - [ ] [QA] Trigger BitRise builds for unit tests and UI tests, in case CHANGELOG conflicts avoid them in GitHub
+ - [ ] [QA] QA Approval
+ - [ ] [DIS] Upload release APK and bundle to internal owncloud instance
+ - [ ] [DIS] Upload & publish release bundle and changelog in Play Store
+ - [ ] [DIS] Update screenshots and store listing, if needed, in Play Store
+ - [ ] [GIT] Create and sign tag `vM.m.p` in HEAD commit of release branch, in owncloud/android
+ - [ ] [GIT] Move tag `latest` pointing the same commit as the release commit
+ - [ ] [GIT] Publish a new [release](https://github.com/owncloud/android/releases) in owncloud/android
+ - [ ] [DIS] Release published in Play Store
+ - [ ] [COM] Inform `#updates` in internal chat that release is out
+ - [ ] [GIT] Fix conflicts if they happen. GitHub option will help and will merge `master` into release branch, getting it ready to fast forward
+ - [ ] [GIT] Merge (not rebase!!!!) `release/M.m.p` branch into `master`, in owncloud/android
```

Comment:
The final step is merging into `master` as every PR, clicking the green button that we find in the PRs. `git merge master` is doable in the previous step, to update the release branch with `master`. You can do it locally and then push to the server, or using the GitHub tools as mentioned in the previous comment. 

Anyway, i will do a little change in the step description. Instead of `Merge (not rebase!!!!) ` i will set `Merge without rebasing` to clarify that merge and rebase are two different things, that we use to use as alternatives but actually they are not.